src/components/Bars/components/BarLoading.tsx

```diff
@@ -0,0 +1,35 @@
+import { cx } from "@/utils/cx/cx";
+import { useGraph } from "@/hooks/use-graph/use-graph";
+import { useId } from "react";
+
+export const BarLoading = () => {
```

Comment:
My fault on this think i told you to name it `BarLoading` just rename it to `BarsVerticalLoading`